cmd/ddev/cmd/composer-create.go

```diff
@@ -62,54 +61,52 @@ ddev composer create --prefer-dist --no-interaction --no-dev psr/log
 			}
 		}
 
+		docRoot := app.GetDocroot()
 		composerRoot := app.GetComposerRoot(false, false)
 
-		// Make the user confirm that existing contents will be deleted
-		util.Warning("Warning: MOST EXISTING CONTENT in the Composer root (%s) will be deleted by the composer create-project operation. Only .ddev, .git and .tarballs will be preserved.", composerRoot)
-		if !composerCreateYesFlag {
-			if !util.Confirm("Would you like to continue?") {
-				util.Failed("create-project cancelled")
-			}
-		}
-
 		err = os.MkdirAll(composerRoot, 0755)
 		if err != nil {
 			util.Failed("Failed to create composerRoot: %v", err)
 		}
 
-		// Remove most contents of Composer root
-		util.Warning("Removing any existing files in Composer root")
-		objs, err := fileutil.ListFilesInDir(composerRoot)
-		if err != nil {
-			util.Failed("Failed to create project: %v", err)
-		}
-
-		for _, o := range objs {
-			// Preserve .ddev, .git, .tarballs
-			if o == ".ddev" || o == ".git" || o == ".tarballs" {
-				continue
+		// If composer root is not the app root, make sure it's empty
```

Comment:
Comment may not say what you want it to? You're saying "fail if it's not empty" right?

Reply:
isn't fail if it's not empty the same as make sure it's empty?

Reply:
Changed it to what you said.